templates/lustrum12/opening.html.twig

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
```

Comment:
Je kan misschien wat common html in een los template stoppen, check https://twig.symfony.com/doc/3.x/templates.html#template-inheritance voor dit.

Reply:
Ik zal komende tijd even kijken of ik dit kan doen. Kan idd wel makkelijk zijn om op die manier de header wat op te schonen. Voor nu heb ik iig de zandloper in een template gezet. Ik hoop dat dat voor nu goed genoeg is.